README.md

```diff
@@ -40,7 +40,7 @@ Available years for census block group and census tract identifiers include 1990
     | Census Tract | State + County + Tract | 2+3+6=11 | Tract 32 in Hamilton County | 39061003200 | 
     | Block Group | State + County + Tract +<br /> Block Group | 2+3+6+1=12 | Block Group 1 in Tract 32 | 390610032001 |
     
-Due to inconsistencies in the 1970 and 1980 tract identifiers, we concatenated the state FIPS (`NHGISST`), county FIPS (`NGHISCTY`), and tract FIPS (the last 4 or 6 digits of `GISJOIN2`) to construct the full `fips_tract_id`. Since the length of tract FIPS codes varied, we padded all tract FIPS to the maximum 6 digits using zeros. 
+*Note that some tract identifiers before 2000 are only 4 digits, instead of 6, resulting in 9-digit `census_tract_id`s.*
```

Comment:
Can we say something more general about the tract/block group identifiers?

Let's define our tract identifier as the concatenation of the state, county, tract, and block group fips identifiers (commonly called `GEOJOIN` or `GEOID` in census data).  Then, we can say that _all_ census tract identifiers are 9 digits and _all_ census block group identifiers are 10 digits _except_ for certain block groups in 1990, 1980, and 1970, in which case tracts will be 8 digits (or 9 digits for block groups within these tracts).

Also is it the case the block group number is always a single digit?  The block group number will never go above 9 to 10? making it 11 instead of 10 digits?  I don't think so, but wasn't sure.

Reply:
Block group is always one digit because, for example, block group 1 contains all blocks 100-199.